inc/extras.php

```diff
@@ -263,3 +263,28 @@ function _s_copyright_year( $atts ) {
 	return esc_html( $args['starting_year'] . $args['separator'] . $current_year );
 }
 add_shortcode( '_s_copyright_year', '_s_copyright_year', 15 );
+
+/**
+ * Get the theme colors for this project. Set these first in the Sass partial then migrate them over here.
+ *
+ * @return array The array of our color names and hex values.
+ * @author Corey Collins
+ */
+function _s_get_theme_colors() {
+	return array(
+		__( 'Alto', '_s' )           => '#ddd',
```

Comment:
@coreymcollins The translation functions should be 

```suggestion
		esc_html__( 'Alto', '_s' )           => '#ddd',
```